src/NzbDrone.Api/Queue/QueueResource.cs

```diff
@@ -17,7 +19,10 @@ public class QueueResource : RestResource
         public decimal Size { get; set; }
         public string Title { get; set; }
         public decimal Sizeleft { get; set; }
+
+        [JsonConverter(typeof(TimeSpanConverter))]
         public TimeSpan? Timeleft { get; set; }
```

Comment:
It looks pretty bad in the UI, but this is pretty ugly, I'm going to see what I can do to improve the formatting of this in the UI, then this isn't required at all.